scripts/deploy-kovan.sh

```diff
@@ -0,0 +1,82 @@
+# USDC-B
+seth send $REGISTRY "add(address)" "0xaca10483e7248453BB6C5afc3e403e8b7EeDF314"
+# WBTC-A
+seth send $REGISTRY "add(address)" "0xB879c7d51439F8e7AC6b2f82583746A0d336e63F"
+# TUSD-A
+seth send $REGISTRY "add(address)" "0xe53f6755A031708c87d80f5B1B43c43892551c17"
+# ZRX-A
+seth send $REGISTRY "add(address)" "0x85D38fF6a6FCf98bD034FB5F9D72cF15e38543f2"
+# KNC-A
+seth send $REGISTRY "add(address)" "0xE42427325A0e4c8e194692FfbcACD92C2C381598"
+# MANA-A
+seth send $REGISTRY "add(address)" "0xdC9Fe394B27525e0D9C827EE356303b49F607aaF"
+# USDT-A
+seth send $REGISTRY "add(address)" "0x9B011a74a690dFd9a1e4996168d3EcBDE73c2226"
+# PAXUSD-A
+seth send $REGISTRY "add(address)" "0x3d6a14C9542B429a4e3d255F6687754d4898D897"
+# COMP-A
+seth send $REGISTRY "add(address)" "0x16D567c1F6824ffFC460A11d48F61E010ae43766"
+# LRC-A
+seth send $REGISTRY "add(address)" "0x436286788C5dB198d632F14A20890b0C4D236800"
+# LINK-A
+seth send $REGISTRY "add(address)" "0xF4Df626aE4fb446e2Dcce461338dEA54d2b9e09b"
+# BAL-A
+seth send $REGISTRY "add(address)" "0x8De5EA9251E0576e3726c8766C56E27fAb2B6597"
+# YFI-A
+seth send $REGISTRY "add(address)" "0x5b683137481F2FE683E2f2385792B1DeB018050F"
+# GUSD-A
+seth send $REGISTRY "add(address)" "0x0c6B26e6AB583D2e4528034037F74842ea988909"
+# UNI-A
+seth send $REGISTRY "add(address)" "0xb6E6EE050B4a74C8cc1DfdE62cAC8C6d9D8F4CAa"
+# RENBTC-A
+seth send $REGISTRY "add(address)" "0x12F1F6c7E5fDF1B671CebFBDE974341847d0Caa4"
+# AAVE-A
+seth send $REGISTRY "add(address)" "0x9f1Ed3219035e6bDb19E0D95d316c7c39ad302EC"
+# UNIV2DAIETH-A
+seth send $REGISTRY "add(address)" "0x03f18d97D25c13FecB15aBee143276D3bD2742De"
+# PAXG-A
+seth send $REGISTRY "add(address)" "0x822248F31bd899DE327A760a78B6C84889aF180D"
+
+# put(bytes32 _ilk, address _join, address _gem, uint256 _dec, uint256 _class, address _pip, address _xlip, string calldata _name, string calldata _symbol)
+
+# RWA001-A                                                                                        bytes32 _ilk,                                          address _join,                               address _gem,                                uint256 _dec, uint256 _class, address _pip, address _xlip, string calldata _name, string calldata _symbol
+seth send $REGISTRY "put(bytes32,address,address,uint256,uint256,address,address,string,string)" $(seth --to-bytes32 "$(seth --from-ascii "RWA001-A")") "0x029A554f252373e146f76Fa1a7455f73aBF4d38e" "0x8F9A8cbBdfb93b72d646c8DEd6B4Fe4D86B315cB" "18" "3" "0x09710C9440e5FF5c473efe61d5a2f14cA05A6752" "0x0" '"RWA001-A: 6s Capital"' '"RWA001-A"'
+# NS2DRP-A
+seth send $REGISTRY "put(bytes32,address,address,uint256,uint256,address,address,string,string)" $(seth --to-bytes32 "$(seth --from-ascii "NS2DRP-A")") "0x4B8C10da2B70dE45f7Ea106A961F2Fb79f5bC2bE" "0x1C3765c94aF9b7eB3fdEC69Eddb7Ddf27f2BcFf4" "18" "3" "0x82a561D6f5013766203776a26123ce5B9389109b" "0x0" '"NS2DRP-A: Centrifuge: New Silver"' '"NS2DRP-A"'
```

Comment:
Nit but if there's a bashy way to get each argument on a separate line that would be much more readable.

At that point, you probably don't need the commented-out signature above either, assuming you keep the per-argument comments.